### PR TITLE
[Spec] Specified the stdlib module `acl`

### PR DESCRIPTION
**Pull request by junkil-park**
### Description

- Added the specs for `acl`
- Enabled an axiom in`from_bcs`. However, two generic axioms still don't work causing a timeout.
- Completed the specs for `any` and `copyable_any`

### CODE REVIEW

File: aptos-move/framework/move-stdlib/sources/acl.move [hunk 1]
```diff
@@ -43,4 +43,16 @@ module std::acl {
...
+    fun verify_remove(acl: ACL, addr: address): ACL {
+        remove(&mut acl, addr);
+        acl
+    }
+
+    #[verify_only]
+    fun verify_add(acl: ACL, addr: address): ACL {
+        add(&mut acl, addr);
+        acl
+    }
```
Comment (meng-xu-cs) [inline aptos-move/framework/move-stdlib/sources/acl.move:57]: These `#[verify_only]` functions might not be needed once #792 lands. I would suggest a review and landing of #792 first and then rebase this PR.

**Review by meng-xu-cs — CHANGES_REQUESTED** [review]

File: aptos-move/framework/move-stdlib/sources/acl.move (hunk 1 above)
Comment (meng-xu-cs) [reply aptos-move/framework/move-stdlib/sources/acl.move:57]: Also, I am expecting `spec verify_remove {}` and `spec verify_add {}` blocks? Or we are just checking data invariants? In the latter case, we don't need these blocks.

File: aptos-move/framework/aptos-stdlib/sources/from_bcs.spec.move [hunk 2]
```diff
@@ -11,16 +11,15 @@ spec aptos_std::from_bcs {
...
         // `deserialize` is an injective function.
-        // TODO: disabled due to the issue with generic axioms.
-        // axiom<T> forall b1: vector<u8>, b2: vector<u8>:
-        //     (deserialize<T>(b1) == deserialize<T>(b2) ==> b1 == b2);
+        axiom<T> forall b1: vector<u8>, b2: vector<u8>:
+            (deserialize<T>(b1) == deserialize<T>(b2) ==> b1 == b2);
 
         // `deserialize` is an inverse function of `bcs::serialize`.
-        // TODO: disabled due to the issue with generic axioms.
+        // TODO: disabled because this generic axiom causes a timeuot.
```
Comment (movekevin) [inline aptos-move/framework/aptos-stdlib/sources/from_bcs.spec.move:18]: nit: timeuot -> timeout. Here and in the next TODO below.

File: aptos-move/framework/move-stdlib/sources/acl.move (hunk 1 above)
Comment (movekevin) [reply aptos-move/framework/move-stdlib/sources/acl.move:57]: The PR you linked to doesn't seem right. Did you mean to link to a PR in the Move repo?

**Review by movekevin — APPROVED** [review]

File: aptos-move/framework/move-stdlib/sources/acl.move (hunk 1 above)
Comment (meng-xu-cs) [reply aptos-move/framework/move-stdlib/sources/acl.move:57]: Yes, sorry, I was meant to link the one in the Move repo

Comment (junkil-park) [reply aptos-move/framework/move-stdlib/sources/acl.move]: @meng-xu-cs , it works great!

**Review by meng-xu-cs — APPROVED** [review]

**Pull request merged**